### PR TITLE
179 consistent highlighting state

### PR DESCRIPTION
**Pull request by mx-e**

### CODE REVIEW

File: src/components/ClusterMap/cluster-map-view.module.css [hunk 1]
```diff
@@ -3,6 +3,14 @@
   width: 100%;
   position: relative;
 }
+
+.unClickDiv{
```
Comment (lillijo) [inline src/components/ClusterMap/cluster-map-view.module.css]: is this still used somewhere?

File: src/store/actions/actions.js [hunk 2]
```diff
@@ -23,88 +24,117 @@ export const timerangeFilterChange = value => {
...
+    type: actionTypes.UNCLICKED
   };
 };
 
 export const fetchClusterData = () => {
   return dispatch => {
-    axios.get("https://localhost/api/clustering").then(result => {
-      dispatch(updateClusterData(result.data));
-      dispatch(processDataIfReady());
+    axios.get("./cluster.json").then(result => {
```
Comment (lillijo) [inline src/store/actions/actions.js]: is your /api/clustering not working?

Comment (mx-e) [reply src/store/actions/actions.js]: nope, don't have the dump  :)

File: src/components/ClusterMap/cluster-map-view.module.css (hunk 1 above)
Comment (mx-e) [reply src/components/ClusterMap/cluster-map-view.module.css]: that's right

Comment (mx-e) [reply src/components/ClusterMap/cluster-map-view.module.css]: not used, ill remove

**Pull request merged**